docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - ./plugins/events:/var/www/html/admin/tool/events
       - ./plugins/study:/var/www/html/mod/study
       - ./plugins/content:/var/www/html/filter/content
+      - ./plugins/direct_events_plugin:/var/www/html/local/direct
```

Comment:
This will have some waterfall implications on code, but for clarity I'd like to change the naming conventions here:

1. Let's put the code into `./plugins/fe_events_direct`. @P-Gill97 can then use `./plugins/fe_events_moodle` for k12-182
2. Let's name the plugin `fe_events_direct` (e.g. map it to `/var/ww/html/local/fe_events_direct` within Moodle)

I'm also open to better naming conventions if others have ideas that will help with clarity.